tests/nested/manual/core20-early-config/defaults.yaml

```diff
@@ -8,3 +8,5 @@ defaults:
     system:
       power-key-action: ignore
       disable-backlight-service: true
+    system:
+      timezone: Europe/Malta
```

Comment:
unclear how this didn't break the test as merged, but the `system` key here is duplicated, making the core20-early-config test fail because it can't build the gadget snap. 

I think this speaks to a larger issue around our nested tests in that it's unclear if when run as a PR they are actually always including snapd from the spread branch in the test. I am led to believe that they do not.

Reply:
to be clear, apparently we set BUILD_SNAPD_FROM_CURRENT to false if it is not already set, but it is set when we run nested spread tests via github actions: 

https://github.com/snapcore/snapd/blob/86feee019283452df4f1dd1bd38364b13ca78a84/spread.yaml#L832

and

https://github.com/snapcore/snapd/blob/86feee019283452df4f1dd1bd38364b13ca78a84/.github/workflows/test.yaml#L281